subgraph-js/src/federation-atlas.ts

```diff
@@ -0,0 +1,62 @@
+import { Atlas, Schema, gql } from "@apollo/core-schema";
+
+export const SUBGRAPH_BASE = Schema.basic(
+  gql `${'builtin:subgraph-base'}
+    @link(url: "https://specs.apollo.dev/federation/v1.0", as: "",
+          import: "@key @requires @provides @external")
```

Comment:
The syntax of this `import` (and the others in that file) is incompatible with the definition of `link/v0.3` made below (`import` is defined of type `[Import]` but that exclude the string syntax). I continue to think that we shouldn't have multiple different syntax for this `import` argument: it's going to confuse users, _especially_ if those different syntaxes are not universally supported.

I'll also note that if we release code as GA that support 2 different syntaxes, it's a one-way door: user my start using both and we can't remove one with breaking those users. It's not a decision we can "postpone".